public/src/components/channelManagement/bannerTests/bannerVariantPreview.tsx

```diff
@@ -9,7 +9,7 @@ import { useModule } from '../../../hooks/useModule';
 import useTickerData, { TickerSettingsWithData } from '../hooks/useTickerData';
 import { mockAmountsCardData, SelectedAmountsVariant } from '../../../utils/models';
 import { BannerDesign, BannerDesignProps } from '../../../models/bannerDesign';
-import { ArticleCounts } from '../epicTests/epicVariantPreview';
+import { ArticleCounts } from '../epicTests/variantPreview';
```

Comment:
perhaps one for another PR, but clearly `ArticleCounts` isn't specific to epics and should be in a shared file

Reply:
ah yes good spot...there is probably some refactoring to consolidate things like this! I'll add something to the back log!